management/src/main/java/nearsoft/skytouch/management/repository/ProductRepository.java

```diff
@@ -0,0 +1,9 @@
+package nearsoft.skytouch.management.repository;
+
+import nearsoft.skytouch.common.model.Product;
+
+import java.util.List;
+
+public interface ProductRepository {
```

Comment:
A good practice is to add comments about what the purpose of the contract and its methods are.

Reply:
Ok, i'll add the comments